apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V110__add_loaded_batch_loaded_file_idx.sql

```diff
@@ -0,0 +1,8 @@
+/* 
+ BFD-1505
+ There are periodic JDBC errors related to the Loaded Filter Manager bloom filter implementation.
+ 
+ This may correct the issue by having postgres not do a sequential scan on loaded_batches with an index.
+ */
+ 
+create index ${logic.index-create-concurrently} if not exists "loaded_batches_loaded_file_id_idx" on "loaded_batches" ("loaded_file_id" DESC NULLS LAST);
```

Comment:
What's with the double-quotes? Don't need these as artifact names adhere to snake-case.

Reply:
migration is failing the ITs so Im trying to find the right syntax to get it to pass. Locally investigating

Reply:
so I built the entire bfd-model with the syntax I showed; the failing ITs are not in bfd-model.

Reply:
I see. For some reason Im only able to get this migration to work with psql logic only.

Reply:
OK, the following should do the trick:

create index ${logic.index-create-concurrently}
if not exists loaded_batches_loaded_file_id_idx on loaded_batches (loaded_file_id DESC ${logic.psql-only} nulls last
);

The problem is that HSQL does not support 'nulls last' so we wrap that part in psql-only; thus we create the index in both hsql & psql with appropriate attributes for both.

Reply:
create index ${logic.index-create-concurrently}
if not exists loaded_batches_loaded_file_id_idx on loaded_batches (loaded_file_id DESC ${logic.psql-only} nulls last
);

You have that exactly like the above 3 lines?

Reply:
All good now thx